Context menu contains/does not contain fails to apply

Version 0.21.1

Given a table with a string column, right click on cell an attempt to use the filter > contains or does not contain 

Expected: filter is applied

Actual: table is stuck loading, error in worker console appears saying:
```
java.lang.RuntimeException: Column "" doesn't exist in this table, available columns: [hvfhs_license_num, dispatching_base_num, originating_base_num, request_datetime, on_scene_datetime, pickup_datetime, dropoff_datetime, PULocationID, DOLocationID, trip_miles, trip_time, base_passenger_fare, tolls, bcf, sales_tax, congestion_surcharge, airport_fee, tips, driver_pay, shared_request_flag, shared_match_flag, access_a_ride_flag, wav_request_flag, wav_match_flag]
```

And browser console says:
```
(dh-internal.js:1 Uncaught Error: java.lang.AssertionError: Table already closed, cannot be used again)
```

All other filters seem to work, just this one is broken

--- a/web/client-api/src/main/java/io/deephaven/web/client/api/filter/FilterValue.java
+++ b/web/client-api/src/main/java/io/deephaven/web/client/api/filter/FilterValue.java
@@ -178,6 +178,7 @@ public class FilterValue {
 
     private FilterCondition makeContains(FilterValue term, double casesensitivity) {
         ContainsCondition contains = new ContainsCondition();
+        contains.setReference(this.descriptor.getReference());
         contains.setSearchString(term.descriptor.getLiteral().getStringValue());
         contains.setCaseSensitivity(casesensitivity);
 

